src/main/java/io/galeb/manager/entity/RuleType.java

```diff
@@ -55,4 +55,8 @@ public RuleType setRules(Set<Rule> rules) {
         return this;
     }
 
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?